install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```
@@ -222,6 +222,12 @@ spec:
         {{- end }}
         {{- if or .Values.prometheus.enabled .Values.hubble.metrics.enabled }}
         ports:
+        {{- if .Values.hubble.peerService.enabled }}
+        - name: prometheus
+          containerPort: {{ .Values.hubble.peerService.containerPort }}
+          hostPort: {{ .Values.hubble.peerService.containerPort }}
```

[This CI failure](https://github.com/cilium/cilium/runs/7034928386?check_suite_focus=true) is legit:

> Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(DaemonSet.spec.template.spec.containers[0].ports[0]): missing required field "containerPort" in io.k8s.api.core.v1.ContainerPort

```suggestion
          containerPort: {{ .Values.hubble.peerService.targetPort }}
          hostPort: {{ .Values.hubble.peerService.targetPort }}
```